public/app/features/alerting/unified/insights/grafana/EvalDurationScene.tsx

```diff
@@ -0,0 +1,36 @@
+import { PanelBuilders, SceneFlexItem, SceneQueryRunner, SceneTimeRange } from '@grafana/scenes';
+import { DataSourceRef, GraphDrawStyle } from '@grafana/schema';
+
+const QUERY_A =
```

Comment:
nit: perhaps we could opt for more descriptive names for these constants instead of using 'QUERY_A' and 'QUERY_B'?

Reply:
Good point, I removed the variable assignments as they were only used once hence not necessary.